### PR TITLE
chart version of local example

### PR DESCRIPTION
**Pull request by fgimenez**
Related to giantswarm/giantswarm#1833 and giantswarm/giantswarm#1853

As in https://github.com/giantswarm/cert-operator/pull/88, follows method 1 from helm's recommendations about TPR https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/third_party_resources.md

### CODE REVIEW

**Comment by puja108** [comment]
not sure I (or the fellow reviewers) can judge this well, could you maybe involve a dev into this review

**Comment by fgimenez** [comment]
@puja108 sure thx, added Tim and Joe

File: examples/local/README.md [hunk 1]
```diff
@@ -104,42 +53,63 @@ running the operator. For Minikube run `eval $(minikube docker-env)` before
...
-Creating ThirdPartyObject should eventually result in a working K8s cluster on
-AWS. To test if the cluster is ready check the operator's pod logs with the
+`aws-resource-lab-chart` accepts the following configuration parameters:
+* `clusterName` - Cluster's name.
+* `commonDomain` - Cluster's etcd and API common domain.
+* `aws.region` - AWS region.
+* `aws.az` - AWS availability zone.
+* `aws.ami` - AWS image to be used on both master and worker machines.
+* `aws.instanceTypeMaster` - Master machines instance type.
+* `aws.instanceTypeWroker` - Worker machines instance type.
```
Comment (JosephSalisbury) [inline examples/local/README.md]: I assume this is a typo.

**Comment by fgimenez** [comment]
Thank you guys, I've tried it locally and seemed to work fine, hopefully we'll see this moving soon :)

**Pull request merged**